compiler/rustc_hir_typeck/src/fn_ctxt/suggestions.rs

```diff
@@ -1,8 +1,6 @@
 use super::FnCtxt;
 
-use crate::errors::{
-    AddReturnTypeSuggestion, ExpectedReturnTypeLabel, SuggestBoxing, SuggestConvertViaMethod,
-};
```

Comment:
Any particular reason to remove those imports?

Reply:
We should be using `errors::` in the code instead of importing them.

Reply:
iirc the main reason being preventing merge conflicts as diagnostic imports tend to get changed a lot

Reply:
Yup